django/utils/six.py

```diff
@@ -852,3 +854,34 @@ def python_2_unicode_compatible(klass):
     else:
         memoryview = buffer
     buffer_types = (bytearray, memoryview)
+
+
+# backport of Python 3.4's glob.escape
```

Comment:
I just didn't want to create a new file for that, but I guess it's better that way.